addons/website_blog/static/src/js/website_blog.js

```diff
@@ -7,7 +7,8 @@ const publicWidget = require('web.public.widget');
 
 publicWidget.registry.websiteBlog = publicWidget.Widget.extend({
     selector: '.website_blog',
-    events: {
+    disabledInEditableMode: false,
+    read_events: {
```

Comment:
The widget, after your commit, broke the main principle. A widget is created for "visit/readonly" mode, destroyed before entering edit mode and potentially recreated for edit mode (if `disabledInEditableMode` is set to false). After your commit, your widget did some things in its `start` method that it did not clean in its `destroy`. Meaning what you had was:

- Create the widget for visit/read mode (for example: remove the `o_container_as_first` classes, attach events to the bus, etc)
- Enter edit mode -> first destroy the widget (-> does nothing)
- The widget is not set as active for edit mode, it is not recreated... but your bus still has attached events etc... so it kinda works.

After my commit, what I am changing is that the widget is also started in edit mode, and properly cleaned when destroyed (meaning restoring the container_as_first classes, detach events, etc). That also means that I had to prevent readonly-behaviors to be active in edit mode: that's what `read_events` is for and also the `if (this.editableMode)` added someplace else.

I agree that in this case, the difference between your implementation and this may seem small but it is more robust :wink: 

Note: another possibility would be to create two widgets instead of one with only the new one being active in edit mode. I went the other way.